Admin/AbstractAdmin.php

```diff
@@ -2845,6 +2852,22 @@ public function getDashboardActions()
     /**
      * {@inheritdoc}
      */
+    public function getSearchResultLink($object)
```

Comment:
A voter does other things. Imagine if you want to render a `edit` link if an object is less than 2 years old and a `show` link for the rest. How can you solve this with a voter?

I've started a **final** discussion for here https://github.com/sonata-project/dev-kit/issues/79